src/main/java/com/api/service/RocketService.java

```diff
@@ -0,0 +1,31 @@
+package com.api.service;
+
+import com.api.dto.RocketDto;
+import com.api.entity.SxRocket;
+import com.api.mapper.RocketMapper;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class RocketService {
```

Comment:
what is the purpose of this service? is it just doing the mapping? you can do that without separation in other Service